pages/news.html

```diff
@@ -88,6 +88,7 @@ <h1>NEWS</h1>
 <div class="container page-wrapper">
     <div class="container">
         <div class="row content-padding subpage-content">
+	  <p><strong>16/02/2024</strong> - Uyuni 2024.02 brings important CVEs fixes. Please, read carefully the <a href='../pages/stable-version.html#releasenotes'>release notes!</a> The Uyuni server container image is now also available for aarch64 (ARM64) and mgradm provides new start/stop/restart commands.</p>
```

Comment:
Same as before, but no need to accept my idea ;-)

Reply:
Already included (and announced) in 2024.01.